### ISSUE TITLE
List of accepted taxa with their synonyms as output

### ISSUE THREAD

**Issue by damianooldoni** [trias-project/unified-checklist#41]
It would be nice to have as output of unified checklist a list of accepted taxa with their synonyms.

**Comment by peterdesmet** [comment]
Indeed, would be useful for e.g. @mcoupremanne to have a wider "net" of names to find non-native occurrences in local databases.

**Comment by damianooldoni** [comment]
This is surely possible within `rgbif` package by using function `name_usage()` and endpoint `synonyms`. Here below a self-contained example based on two accepted taxa (input). Notice how output is a data.frame, in the example below called `synonyms`, with all synonyms. The taxon keys of accepted taxa are used as input and are returned back in column `acceptedKey` of `synonyms`.

```r
library(tidyverse)
library(rgbif)

accepted_taxa <- tibble(
  key = c(1000046, 2427148), # GBIF Backbone keys
  scientificname = c(
    "Acidianus brierleyi (Zillig & al., 1980) Segerer & al., 1986",
    "Lithobates megapoda (Taylor, 1942)"
  )
)

synonyms <- map_dfr(
  accepted_taxa$key, 
  function(taxon) {
    name_usage(key = taxon, data = "synonyms", return = "data")
  }
)
```

I hope this chunk of code can help. For any coding problem related to this issue, just let me know.

**Comment by peterdesmet** [comment]
This was implemented in #42 and now available at https://github.com/trias-project/unified-checklist/blob/master/data/interim/taxa_unified_and_synonyms.csv. Closing issue.
